bedrock/firefox/urls.py

```diff
@@ -79,9 +73,6 @@
     url(r'^firefox/ios/testflight', views.ios_testflight, name='firefox.ios.testflight'),
     page('firefox/mobile-download', 'firefox/mobile-download.html'),
     page('firefox/mobile-download/desktop', 'firefox/mobile-download-desktop.html'),
-    page('firefox/private-browsing', 'firefox/private-browsing.html', alt_canonical_paths={
```

Comment:
Searching through the repo I can still find several references to `{{ url('firefox.private-browsing') }}` which will need updating.

Reply:
Got 'em.